__init__.py

```diff
@@ -109,6 +109,20 @@ def _find_entity(self, entity, domains):
                               "dev_name": entity})
         return ha_entity
 
+    # Routine for entiti availibility check
+    def _check_availability(self, ha_entity):
+        """ Simple routine for checking availability of entity inside
+        Home Assistent. """
+
+        if ha_entity['state'] == 'unavailable':
+            """ Check if state is `unavailable`, if yes, inform user about it. """
+
+            self.speak_dialog('homeassistant.device.unavailable', data={
+                            "dev_name": ha_entity['dev_name']})
+       """ Return result to underliing function. """
```

Comment:
This line needs to match the indent of the IF statement. It throws an indentation error